src/AzureClient/AzureClient.cs

```diff
@@ -522,6 +523,18 @@ public async Task<ExecutionResult> SetActiveTargetAsync(IChannel channel, string
             return AvailableTargets.First(target => target.TargetId == ActiveTarget.TargetId).ToExecutionResult();
         }
 
+        private static IEnumerable<string> ReadAllLines(Stream stream)
+        {
+            using (var reader = new StreamReader(stream))
+            {
+                string line = "";
```

Comment:
nit: I usually find `string.Empty` clearer and less error-prone.

Reply:
done